workflow/config/config.yaml

```diff
@@ -12,69 +21,108 @@ train_seeds:
   - 7
   - 8
   - 9
+
+# Seeds used when sampling molecules from the trained models.
+# The amount of 'sample_seeds' parameter specifies how many times the 'sample_molecules_RNN' step is executed,
+# each time using the same trained model but with different random seed values.
 sample_seeds:
   - 0
+
+# Specifies by how many times the input data is augmented (or enumerated) before training.
+# This utilizes the fact that a single molecule can have multiple SMILES representation.
+# For example, a value of 0 means no augmentation, while 100 means each molecule can have
```

Comment:
might be nice to explain how '0' is different from '1'. Also, is it "up to 100" or "exactly 100"? Might be worthwhile looking at the implementation.